### PR TITLE
Don't rewrite template url attributes by default

### PR DESCRIPTION
**Pull request by usergenic**
Polymer 2.0 + HTML Imports polyfill etc doesn't rewrite urls inside `<template>` tags.  So Bundler won't unless a new `rewriteTemplateUrlAttrs` option is provided as `true`.

 - [x] CHANGELOG.md has been updated
 - Fixes https://github.com/Polymer/polymer-bundler/issues/441

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   use new public method `generateManifest()`.
 - Fixed an issue where an immediate `<style>` child of `<dom-module>` was
   not moved into generated `<template>`.
+- BREAKING: Added option `rewriteTemplateUrls` to support old-style rewriting
```
Comment (FredKSchott) [inline CHANGELOG.md]: could you explain what this means, for anyone who doesn't know about old-style?

Comment (usergenic) [inline CHANGELOG.md]: Good idea.  Done.  I also changed the name now to `rewriteUrlsInTemplates` which I think is more intention-revealing.

**Comment by usergenic** [comment]
> Man, rewriteTemplateUrlAttrs really gets passed around from function to function. Would it be possible to support this functionality via a separate function call / mutation step instead?

In some ways it is a by-product of the dom5 API, being that query operations are provided an option to descend into `<template>` child nodes or not, which is essentially what `true` means in the case of `rewriteUrlsInTemplates` (formerly `rewriteTemplateUrlAttrs`).

While it is a flag that gets passed down, I think if we didn't work it that way, we'd need to basically just have conditionals and multiple additional call-sites to the various called rewrite functions in import-utils.

I agree that in the case of `inlineHtmlImport` its yet-another-param that feels like baggage and explosion of arity, but the behavior is significant and it can't be run after the inlining takes place.  In the case of functions like `rewriteAstToEmulateBaseTag` or `rewriteAstBaseUrl` I think it fits too.

**Comment by usergenic** [comment]
@FredKSchott short answer: probably not without significant rework of the way these functions are composed and the benefit would be dubious.

**Comment by usergenic** [comment]
per @valdrinkoshi comment https://github.com/Polymer/polymer-bundler/issues/441#issuecomment-293414941 I deem this PR mergeable as-is.

**Pull request merged**